### DISCUSSION TITLE
more than one backup location

### DISCUSSION THREAD

**Discussion by Masterst0ffel** [keepassxreboot/keepassxc#8852] [Q&A]
Is it possible to select more than one backup location by setting parameters like. 

C:\Users\Sample1\Desktop\ ; D:\Users\Sample2\Desktop\

**Comment by phoerious** [comment]
No. The "backup" file that is created is not meant as a real backup. Only as a precaution if something goes wrong with the original file. You still need to back up files properly to other locations.
